### ISSUE TITLE
ext- packages need to be ignored outright for scanning

### ISSUE THREAD

**Issue by AshleyPinner** [icanhazstring/composer-unused#31]
**Describe the bug**
Consider this example case:

```JSON
{
    "require": {
        "ext-json": "*",
        "ext-pdo": "*",
        "ext-zlib": "*",
        "ext-soap": "*",
        "ext-zip": "*",
        "ext-simplexml": "*",
        "ext-dom": "*",
        "ext-curl": "*",
        "ext-calendar": "*",
        "ext-fileinfo": "*",
        "ext-iconv": "*",
        "ext-mcrypt": "*",
        "ext-openssl": "*",
        "ext-ctype": "*"
    }
}
```

Output:

```
Ignored packages
 ○ ext-json (Unable to locate package)
 ○ ext-pdo (Unable to locate package)
 ○ ext-zlib (Unable to locate package)
 ○ ext-soap (Unable to locate package)
 ○ ext-zip (Unable to locate package)
 ○ ext-simplexml (Unable to locate package)
 ○ ext-dom (Unable to locate package)
 ○ ext-curl (Unable to locate package)
 ○ ext-calendar (Unable to locate package)
 ○ ext-fileinfo (Unable to locate package)
 ○ ext-iconv (Unable to locate package)
 ○ ext-mcrypt (Unable to locate package)
 ○ ext-openssl (Unable to locate package)
 ○ ext-ctype (Unable to locate package)
```

**Additional information**
Generally these directives should just be ignored for the context of scanning. (unless you want to scan for their actual function usages? I suspect not!) - as in they shouldn't even turn up in the `Ignored packages` section as they're `ext-` requirements.

**Comment by icanhazstring** [comment]
Hi @AshleyPinner thanks for the request.
I think in the first iteration ignoring them silently would be a nice addition.

> unless you want to scan for their actual function usages? I suspect not!
In the long run, I want to try to also add static analysis if you are even using this extension inside your code. Like there is already a plugin for PHPStorm which checks that - but would be nice to have it here 😉

**Comment by icanhazstring** [comment]
Solved in version `0.5.0`.
I created an feature proposal for scanning extensions. Give it a thumbs up if this should be implemented :)
